src/containers/Tenant/Diagnostics/Consumers/Consumers.tsx

```diff
@@ -17,18 +19,32 @@ const b = block('ydb-consumers');
 
 interface ConsumersProps {
     path: string;
+    pathType?: EPathType;
+    schemaNestedChildrenPaths?: string[];
 }
 
-export const Consumers = ({path}: ConsumersProps) => {
+export const Consumers = ({path, pathType, schemaNestedChildrenPaths}: ConsumersProps) => {
     const dispath = useDispatch();
 
+    const isCdcStream = pathType === EPathType.EPathTypeCdcStream;
+
+    const targetPath = useMemo(
+        () =>
+            // Since every CDC Stream have only one nested PersQueueGroup, we select only first item
+            (isCdcStream ? schemaNestedChildrenPaths && schemaNestedChildrenPaths[0] : path) ||
```

Comment:
this component looks as it is not sure how to behave

either it should know that it works with CDCStreams and PQGroups, and accept a path, selecting required data from store by itself
or it should accept a final path that it would use to make an api call

but right now it requires a selector outside, having at the same time a hardcoded logic for specifically cdcstream inside

let it be one way or another, not both